comps/embeddings/predictionguard/docker/Dockerfile

```diff
@@ -0,0 +1,30 @@
+
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+FROM python:3.11-slim
+
+ARG ARCH="cpu"
+
+RUN apt-get update -y && apt-get install -y --no-install-recommends --fix-missing \
```

Comment:
We should combine all the RUN commands into a single RUN command. This reduces the number of "layers" in the docker image.